### PR TITLE
Added Bamboo Documentation

### PR DESCRIPTION
**Pull request by AutomationReddy**
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ x] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x ] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ x] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: docs/BambooIntegration.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+---
+id: bamboo
+title: Bamboo Integration
+---
+
+WebdriverIO offers a tight integration to CI systems like [Bamboo](https://www.atlassian.com/software/bamboo). With the `junit | allure` reporters, you can easily debug your tests as well as keep track of your test results. The integration is pretty easy.
```
Comment (christian-bromann) [inline docs/BambooIntegration.md:6]: How about we link the reporter:
```suggestion
WebdriverIO offers a tight integration to CI systems like [Bamboo](https://www.atlassian.com/software/bamboo). With the [JUnit](https://webdriver.io/docs/junit-reporter.html) or [Allure](https://webdriver.io/docs/allure-reporter.html) reporter, you can easily debug your tests as well as keep track of your test results. The integration is pretty easy.
```

File: docs/BambooIntegration.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+1. Install the `junit` test reporter: `$ npm install @wdio/junit-reporter --save-dev`)
+1. Update your config to save your XUnit results where Bamboo can find them, (and specify the `junit` reporter):
```
Comment (christian-bromann) [inline docs/BambooIntegration.md:9]:
```suggestion
1. Update your config to save your JUnit results where Bamboo can find them, (and specify the `junit` reporter):
```

File: docs/BambooIntegration.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
+---
+id: bamboo
+title: Bamboo Integration
+---
+
+WebdriverIO offers a tight integration to CI systems like [Bamboo](https://www.atlassian.com/software/bamboo). With the `junit | allure` reporters, you can easily debug your tests as well as keep track of your test results. The integration is pretty easy.
+
+1. Install the `junit` test reporter: `$ npm install @wdio/junit-reporter --save-dev`)
```
Comment (christian-bromann) [inline docs/BambooIntegration.md:8]: Let's stay consistent and call it JUnit
```suggestion
1. Install the JUnit test reporter: `$ npm install @wdio/junit-reporter --save-dev`)
```

File: docs/BambooIntegration.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+        },
+        outputFileFormat: function (options) {
+            return `results-${options.cid}.xml`;
+        }]
+    ],
+    // ...
+}
+```
+
+The reports will be similar for all the frameworks and you can use anyone ```Mocha | Jasmine | Cucumber```.
```
Comment (christian-bromann) [inline docs/BambooIntegration.md:43]:
```suggestion
The reports will be similar for all the frameworks and you can use anyone: Mocha, Jasmine or Cucumber.
```

File: docs/BambooIntegration.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    > Create a new plan, link your repository (make sure it always points to newest version of your repository) and create your stages
+
+    ![Plan Details](/img/bamboo/plancreation.png "Plan Details")
+
+    I will go with the default stage and job. In your case, you can create your own stages and jobs
+
+    ![Default Stage](/img/bamboo/defaultstage.png "Default Stage")
+
+
+
```
Comment (christian-bromann) [inline docs/BambooIntegration.md:59]:
```suggestion

```

**Review by christian-bromann — CHANGES_REQUESTED** [review]
Some minor comments:

**Review by christian-bromann — APPROVED** [review]
👍  thanks!

**Comment by AutomationReddy** [comment]
> 
> 
> 👍 thanks!

@christian-bromann Thanks for your support. This is my 1st contribution to the open source project. But definitely not the last one. More to go here.....

**Comment by christian-bromann** [comment]
@AutomationReddy it seems like you made a commit with a different name (`vveeramgari`). You have to sign the CLA again then.

**Comment by AutomationReddyOfficial** [comment]
> 
> 
> @AutomationReddy it seems like you made a commit with a different name (`vveeramgari`). You have to sign the CLA again then.

@christian-bromann I have already signed with the github account. `vveeramgari `is my different account. Any suggestions? I tired logging in with `vveeramgari `account and when I click on the signing, it shows that I have already signed.

**Comment by christian-bromann** [comment]
@AutomationReddyOfficial I would say, merge all these changes into one commit and force push to the branch

**Comment by AutomationReddy** [comment]
@christian-bromann I think I cannot proceed further from this branch as the initial commit is with different user and the signing is not happening with that user. Can I go ahead and raise a fresh PR for you?

**Comment by AutomationReddy** [comment]
@christian-bromann It's done. Thanks

**Pull request merged**